packages/commercetools/helpers/src/index.ts

```diff
@@ -1 +1,15 @@
-console.log('Hello commercetools helpers')
\ No newline at end of file
+import { UiMediaGalleryItem, UiProductName, UiProductPrice } from '@vue-storefront/interfaces'
```

Comment:
You're missing `getVariant` which is pretty important since rest of the operations is on variants